src/components/layout/Modalize.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { ComponentProps, forwardRef } from 'react'
+import { Modalize as RNModalize } from 'react-native-modalize'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+
+const Modalize = forwardRef(function Modalize({ children, ...props }: ComponentProps<typeof RNModalize>, ref) {
```

Comment:
NIT: we could use an arrow function notation.

Reply:
When we converting it to arrow function we get this eslint error:


> Component definition is missing display name (eslint [react/display-name](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/display-name.md))

To avoid it, we either disable the rule or need to manually set the display name with:

```ts
Modalize.displayName = 'Modalize'
```

The complete code would look like this:

```ts
const Modalize = forwardRef<RNModalize, ComponentProps<typeof RNModalize>>(({ children, ...props }, ref) => {
   ...
})
Modalize.displayName = 'Modalize'
```

I personally prefer the named function:

```ts
const Modalize = forwardRef(function Modalize({ children, ...props }: ComponentProps<typeof RNModalize>, ref) {
  ...
})
```

Reply:
Alright!